packages/superset-ui-demo/storybook/stories/superset-ui-chart/ChartDataProviderStories.tsx

```diff
@@ -46,14 +45,19 @@ const FORM_DATA_LOOKUP = {
 
 export default {
   title: 'Core Packages|@superset-ui/chart',
+  decorators: [
+    withKnobs({
+      escapeHTML: false,
+    }),
+  ],
 };
 
 export const dataProvider = () => {
-  const host = text('Set Superset App host for CORS request', 'localhost:9000');
+  const host = text('Set Superset App host for CORS request', 'localhost:8088');
```

Comment:
Use the default backend host since the storybook demo doesn't really need Superset frontend